### ISSUE TITLE
Pylint error on default value

### ISSUE THREAD

**Issue by thorstenkampe** [jeffrimko/Qprompt#13]
Pylint gives an error on default values like:
```python
qprompt.ask_yesno(default = 'n')
```
```
file:1:18: E1123: Unexpected keyword argument 'default' in function call (unexpected-keyword-arg)
```
Same goes for `dft` as keyword.

**Comment by jeffrimko** [comment]
Yeah, that's a result of how the keyword arguments are being handled at the moment. I can look into how to prevent these lint issues.

**Comment by jeffrimko** [comment]
Okay, this issue should be fixed in `0.15.3`. If you installed Qprompt with pip, run `pip install --upgrade qprompt`.
